Design.md

```diff
@@ -313,3 +314,47 @@ The `fcos-builds` bucket, fronted by http://builds.coreos.fedoraproject.org/ wil
 The artifacts under e.g. `30.1234-5/x86_64/` come directly from [coreos-assembler](https://github.com/coreos/coreos-assembler). The `/streams/*.json`, `release.json`, and `releases.json` are higher-level generated metadata objects. See [#98](https://github.com/coreos/fedora-coreos-tracker/issues/98) and [#207](https://github.com/coreos/fedora-coreos-tracker/pull/207) for more information about those.
 
 The stream metadata format (under `/streams`) is intended to be stable, and stream metadata objects will contain links to artifacts in the release bucket. *Everything else about the bucket layout, including its directory structure and the formats of other metadata objects, is subject to change without notice. Third-party tooling should not rely on this structure, and should instead read metadata and artifact URLs directly from stream metadata at the officially documented URL*.
+
+## Version numbers
+
+Originally discussed in [#81](https://github.com/coreos/fedora-coreos-tracker/issues/81) and [#211](https://github.com/coreos/fedora-coreos-tracker/issues/211).
+
+Fedora CoreOS versions will have the form `X.Y.Z.A`:
+
+- X is the Fedora major version, e.g. `30`.
+- Y is the datestamp that the package set was snapshotted from Fedora, e.g. `20191014`.  For mechanical streams, this is the build date.  For development and production streams, it's the date of the snapshot that was promoted.
+- For official builds, Z is a code number corresponding to the stream:
+
+Stream | Z version
+-- | --
+next | 1
+testing | 2
+stable | 3
+next-devel | 10
+testing-devel | 20
+rawhide | 91
+branched | 92
+bodhi-updates-testing | 93
+bodhi-updates | 94
+
+For developer builds (those not produced by the official pipeline), Z is always `dev`.
```

Comment:
Minor: an example-entry for this case would be helpful for people just skimming though that table.